### PR TITLE
Eventbrite Ticket correct order and Required Started Date

### PR DESCRIPTION
**Pull request by bordoni**
_Ref:_ [C#121979](http://central.tri.be/issues/121979)
---

### CODE REVIEW

File: src/resources/js/aggregator-fields.js [hunk 1]
```diff
@@ -614,13 +614,6 @@ tribe_aggregator.fields = {
 			data: rows
 		};
 
-		// if eb then reverse the order of events
-		if ( is_eventbrite ) {
-			args.order = [
-				[ 1, 'desc' ]
-			];
-		}
```
Comment (ggwicz) [inline src/resources/js/aggregator-fields.js]: do you need to run `gulp` to minify/process this change?

Comment (ggwicz) [reply src/resources/js/aggregator-fields.js]: also, does this change need a readme entry?

**Comment by juanfra** [comment]
@bordoni No changelog entry?

**Comment by bordoni** [comment]
True. Changelog.

File: src/resources/js/aggregator-fields.js (hunk 1 above)
Comment (bordoni) [reply src/resources/js/aggregator-fields.js]: No gulp required. Bot will do that

**Comment by bordoni** [comment]
Ready for another pair of eyes

**Pull request merged**